README.md

```diff
@@ -117,6 +117,33 @@ Password for user postgresql: # postgres
 aerogear_mobile_metrics=> select * from mobileappmetrics;
 ```
 
+### Using Swagger UI
+
+If you have the db & server running locally, you can generate and use a swagger spec as follows:
+
+```bash
+go generate ./cmd/metrics-api/metrics-api.go
```

Comment:
Would be nice to add to the `Makefile`.

This would require installing the go-swagger binary, right? I'm not sure what's the best practice for getting it via `dep` or if it even supports it (equivalent of npm's `devDependencies`).